<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>901</width>
     <height>751</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>881</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vbox">
     <item>
      <layout class="QHBoxLayout" name="hbox_xy">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_xy">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>(x,y) = (0,0), 0 points plotted</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hbox_slider">
       <item>
        <widget class="QPushButton" name="btn">
         <property name="text">
          <string>외쳐! 푸리에 도란스포메이션!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_N">
         <property name="text">
          <string>2N-1 = 49</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slider">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>21</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>401</width>
     <height>751</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>381</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_INDEX1">
       <property name="text">
        <string>INDEX1 = 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_INDEX1">
       <property name="minimum">
        <number>-20</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_INDEX2">
       <property name="text">
        <string>INDEX2 = -1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_INDEX2">
       <property name="minimum">
        <number>-20</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
